### ISSUE TITLE
Unsupported flow type FlowType(type=IndexedAccessType)

### ISSUE THREAD

**Issue by toshi-toma** [zxbodya/flowts#44]
Thanks for creating this awesome tool!!

I have some codes includes IndexedAccessType.

But, `flowts` seem to does not support the IndexedAccessType.
Is there a workaround?

```js
// @flow

type Foo = {
  id: string,
  name: string,
}

type IndexedFoo = {
  id: Foo['id'], // IndexedAccessType
  name: Foo['name'], // IndexedAccessType
}

```

run flowts

```
✖ error while trying to convert
Error: sample/test.js: Unsupported flow type FlowType(type=IndexedAccessType)
    at convertFlowType (/node_modules/flowts/node_modules/@zxbodya/babel-plugin-flow-to-typescript/dist/converters/convertFlowType.js:302:11) {
  code: 'BABEL_TRANSFORM_ERROR'
}
```

**Comment by zxbodya** [comment]
It is relatively new flow feature, previously the only option was `$PropertyType<T, 'k'>` or `$ElementType<T, 'k'>`.

Should not be hard to add support for it - will look into this.

**Comment by zxbodya** [comment]
published `flowts@0.15.19` including support for IndexedAccessType

**Comment by toshi-toma** [comment]
Thank you for the quick release!!
